types/index.ts

```diff
@@ -52,3 +52,29 @@ export interface GithubFileData {
   contents: string
   html?: string
 }
+
+export interface GithubLabel {
+  id: number
+  node_id: string
+  url: string
+  name: string
+  color: string
+  default: boolean
+  description: string
+}
+
+export interface GithubIssue {
+  id: number
+  node_id: string
+  number: number
+  title: string
+  user: GithubUser
+  labels: GithubLabel[]
```

Comment:
I think it would be more cache efficient if not resolving sub keys (user, label) and use an id/number to link or simplify to string[] (label name, user name)